### ISSUE TITLE
Improve contrast ratio of the app colors

### ISSUE THREAD

**Issue by mmbotelho** [okTurtles/group-income#565]
### Problem

At this moment, almost all our colors are failing [WCAG 2.0 contrast ratio tests](https://www.w3.org/TR/WCAG/#contrast-minimum). Here are the results for some of the tests I ran:
- [`.button` (gray text over white background)](https://contrast-ratio.com/#%237A7A7A-on-white);
- [`.button .is-primary` (white text over blue background)](https://contrast-ratio.com/#white-on-%235DC8F0);
- [`.button .is-success .is-outline` (green text over white background)](https://contrast-ratio.com/#%23A0D10E-on-white);
- [`.link` (blue text over white background)](https://contrast-ratio.com/#%2317B0EA-on-white)

### Solution

The colors of the brand (the logo and other non-interactive elements, like illustrations) can remain the same, but everything that is part of the actual UI needs a new color palette. We can use colors in the same "family", but we definitely need to make them darker in order to improve our contrast ratio score.

Google's [Material color tool](https://material.io/tools/color) might be helpful in this task.

**Comment by mmbotelho** [comment]
Closing because this issue has been solved with the new UI. 

https://www.figma.com/file/mxGadAHfkWH6qApebQvcdN/Group-Income-2.0
